build-pipelines-PR.yml

```diff
@@ -103,7 +69,6 @@ jobs:
       restoreArguments: -r win-x64
       selectOrConfig: config
       nugetConfigPath: src/nuget.config
-      externalEndpoints: dba0ba50-2d4e-4f12-9f5a-42a638da803b
```

Comment:
is this removal intentional, if so why is this not needed now?

Reply:
here is the reference to the document 
![image](https://user-images.githubusercontent.com/105889062/224370297-3cde593e-1907-4748-85fd-4206496a5a50.png)